src/linker/Linker.Dataflow/MethodBodyScanner.cs

```diff
@@ -1059,6 +1067,10 @@ internal MultiValue DereferenceValue (MultiValue maybeReferenceValue, Dictionary
 					break;
 				case ReferenceValue referenceValue:
 					throw new NotImplementedException ($"Unhandled dereference of ReferenceValue of type {referenceValue.GetType ().FullName}");
+				// Incomplete handling for ref values
+				case FieldValue fieldValue:
```

Comment:
Yeah, it's probably not necessary, but I thought it might be nice to have the case just to point out that it's not fully / correctly implemented. I can remove it if you feel like it's just noise.